Cargo.toml

```diff
@@ -20,8 +20,6 @@ required-features = ["default-no-api-client"]
 split-debuginfo = "unpacked" # Faster debug builds on macOS
 
 [profile.release]
```

Comment:
Maybe we could add a comment here too that the flags are different in CI / for real releases?

Reply:
Might be worth including we saw minimal performance reduction without these (~5% iirc?) which is suitable for local perf analysis.

Reply:
Addressed in c1aed11.